pom.xml

```diff
@@ -219,6 +220,26 @@
                     <check />
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>${version.plugin.jacoco}</version>
+                <configuration>
+                    <includes>
+                        <include>net/bytebuddy/**</include>
```

Comment:
Two stars (`**`) not a mistake here, but a workaround for inconsistency between specification of includes/excludes for instrumentation and for report generation - https://github.com/jacoco/jacoco/issues/34 Maybe be better to add an explicit comment in code here.